src/Tasks/Exec.cs

```diff
@@ -336,17 +336,23 @@ protected override bool HandleTaskExecutionErrors()
         {
             if (IgnoreExitCode)
             {
-                Log.LogMessageFromResources(MessageImportance.Normal, "Exec.CommandFailedNoErrorCode", Command, ExitCode);
+                // Don't log when EchoOff and IgnoreExitCode.
+                if (!EchoOff)
+                {
+                    Log.LogMessageFromResources(MessageImportance.Normal, "Exec.CommandFailedNoErrorCode", Command, ExitCode);
```

Comment:
Hmm. Is it worth preserving this "exited with code" message if we use the scrubbed command from below? It _may_ be useful information, depending on situation--if it's an "always fails with the same code, just go with it" thing you probably don't need that; if it's a "sometimes fails so this is just best effort" it might help chase down a later failure caused by the failure.

Reply:
I omitted the logging because it enables the user to take action on its own based on the output ExitCode of the Exec task.
This could be custom messages, or other a fail back action.

I'm fine changing this to use the "..." placeholder.

Or we can lower the importance when EchoOff (in addition to using the "...").

@rainersigwald what do you prefer?

Reply:
I think as-is is ok; we'll see if anyone complains.